react/Wrapper.tsx

```diff
@@ -22,11 +22,7 @@ const Wrapper: FC<Props> = ({
   showToast,
   customOneClickBuyLink,
 }) => {
-  const productContext: Maybe<ProductContextState> = useProduct()
-
-  if (!productContext) {
-    throw new Error('useProduct must be used within a ProductContextProvider')
-  }
+  const productContext = useProduct()
```

Comment:
Why remove the typing here and the check for an `undefined` `productContext` below?

Reply:
don't you think this check should happen *inside* the `useProduct` hook? I don't think the users of the hook should be concerned with showing this kind of error